### ISSUE TITLE
Paystack Channels 

### ISSUE THREAD

**Issue by kelvinyashim** [binemmanuel/flutter_paystack_max#8]
I removed some channels but still yet they show 
what do i do??

**Comment by binemmanuel** [comment]
Yeah, this is an issue with paystack. You'll have to uncheck them from your paystack dashboard.
